README.md

```diff
@@ -190,6 +190,17 @@ If you somehow need to stick to the current version, please make sure to execute
 ```shell
 go mod tidy
 ```
+### Qitmeer gets stuck on synchronization.
+For some reasons, all your peers are disconnected. For example, all your peers are busy, so your synchronization
+requests to them would exceed the timeout, leading your nodes disconnection from those peers. One recommended
+practice to mitigate this issue is to enhance your connectivity by reporting your external ip (if you have) to the public, 
+so new peers will reach to you even all your peers are disconnected.
+Please see Qitmeer/qitmeer/issues/444 to get more details
```

Comment:
Please fix the link

Reply:
fixed https://github.com/Qitmeer/qitmeer/pull/445/commits/52744e4f9de6fb2d370d9ab0c0491a6110434c08